test/format/salvage.c

```diff
@@ -150,16 +151,15 @@ wts_salvage(void)
     free(cmd);
 
     /* Salvage, then verify. */
-    wts_open(g.home, true, &g.wts_conn);
+    wts_open(g.home, true, &g.wts_conn, false);
     salvage();
     wts_verify("post-salvage verify");
     wts_close();
 
     /* Corrupt the file randomly, salvage, then verify. */
     if (corrupt()) {
-        wts_open(g.home, true, &g.wts_conn);
+        wts_open(g.home, true, &g.wts_conn, true);
         salvage();
-        wts_verify("post-corrupt-salvage verify");
```

Comment:
As verification is almost guaranteed to panic after the corruption is salvaged, due to it retrieving the datafile entry associated with the history store entry there isn't any point verifying here unless we plan to do something other than panic.